lib/moonbeam.js

```diff
@@ -103,21 +103,24 @@ class Moonbeam {
 
     app.options('*', cmw)
 
+    app.get('/competition/leaderboard/vol', cmw, privRoutes.onCompetitionVolLbHttpRequest.bind(privRoutes))
+    app.post('/competition/signup', cmw, privRoutes.onCompetitionSignUpHttpRequest.bind(privRoutes))
```

Comment:
we shoudl either go for binding in the handlers, like you did in the other code at L115, or do the bind call here.

Reply:
Did constructor binding for public routes only, wanted to check what you think